LiskHQ/lisk-hub
# Add followed accounts to dashboard - Closes #903 #890 #905

### What was the problem?
Followed accounts components were not implemented yet

### How did I fix it?
- Added the list view and basic structure so far

<img width="455" alt="screen shot 2018-06-13 at 18 04 32" src="https://user-images.githubusercontent.com/9592216/41363570-874d9c42-6f34-11e8-817f-32b9725b7f03.png">

### How to test it?
- Add some accounts
- Refresh, check that they are still there
- Make sure balance updates on adding/new block/network change

### Review checklist
- The PR solves #903 #890 #905
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
